.circleci/config.yml

```diff
@@ -7,6 +7,11 @@ jobs:
     docker:
       # specify the version
       - image: circleci/golang:1.7.5
+      - image: mysql
```

Comment:
We use mysql 5.5 in production, would it make sense to change this to specify 5.5?